### ISSUE TITLE
Seeds in workspaces are not initialized when workspace is added

### ISSUE THREAD

**Issue by SeriousBug** [dendronhq/dendron#1129]
## Please select the area your Bug applys to. (Multiple selections are Possible)

- [x] Onboard. Initial vault setup and import/export pods 
- [ ] Create. Note creation, lookup, snippets and templates
- [ ] Retrieve. Backlinks, references, graph view
- [ ] Structure. Refactoring, multi-vault and schemas
- [ ] Publish. Sharing your repo with the world
- [ ] Misc (Choose this if your not sure)

## Describe the bug

When a workspace containing seeds is added as a remote vault (workspace vault), the vaults in that workspace are not initialized (the `seeds` folder is not created, and the seed vaults are not pulled).

## To Reproduce

1. Initialize a new Dendron workspace, and add seeds to it. Put this workspace on git. (for example, our internal `org-workspace`)
2. Initialize another Dendron workspace. In this workspace, use the "Vault Add" command with the "remote" option, and enter the git url of the first workspace.

At this point, an error will be thrown that "seeds/seed.vault/root.schema.yml was not found". Checking the folder will reveal that the `seeds` folder is missing. The initialization otherwise works fine, and seeds from the first workspace are added to the `dendron.yml` of the second workspace.

As a workaround, opening the remote workspace that was cloned into the second workspace (`second-ws/first-ws/dendron.code-workspace`) allows Dendron to correctly initialize the seed vaults inside. After that point, reopening the second workspace functions correctly and all vaults and notes are present.

## Expected behavior

The seed vaults should be initialized correctly.

## Desktop (please complete the following information)

- OS: Linux
- Version: 0.54

**Comment by kevinslin** [comment]
Yep, this is currently missing functionality. Adding it to the backlog
